build.sh

```diff
@@ -184,6 +186,10 @@ while [[ $# -gt 0 ]]; do
         ICU_PATH="-DNO_ICU_PATH_GIVEN_SH=1"
         ;;
 
+    --no-jit)
+        NO_JIT="-DNO_JIT=1"
```

Comment:
This works, thanks! I'm using `NO_JIT` in place of `NO_JIT_SH` as `NO_JIT` is not needed afterwards.